distributed/tests/test_client.py

```diff
@@ -3530,21 +3530,25 @@ def test_close_idempotent(c):
     c.close()
 
 
-@nodebug
+@pytest.mark.repeat(1000)
```

Comment:
ooops, no wonder the builds take an hour to finish. At least now I am confident the fix works :D